.github/workflows/build.yml

```diff
@@ -2,12 +2,15 @@ name: Build
 
 on: [ push, pull_request ]
 
+# Declare default permissions as read only.
```

Comment:
This file has been edited across multiple commits with instances of code being removed and then re-added. To streamline the review, the offending commits should be removed, either by rebasing or squashing them into a single, meaningful commit.